src/reducers/app/thunks.ts

```diff
@@ -22,6 +30,31 @@ const getConfig = val => {
   return configs.find(conf => conf.value === val);
 };
 
+const getAttributesForQuery = (attributes, entity) => {
+  if (entity === 'blocks') {
```

Comment:
Why is this condition necessary?

Reply:
because on initial fetch there's no attributes to be shown for 'blocks'. So this function helps satisfy that edge case, and also facilitate getting attributes for the other tabs as well

Reply:
I'm open to alternate suggestions though